libs/ui/components/contents/badge/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { VStack } from '@chakra-ui/react';
+import COLORS from '../../../constants/colors';
+import Text from '../text';
+import FONTS from '../../../constants/fonts';
+import IBadgeProps from './props';
+
+const Badge = ({isSelected, notificationNumber}: IBadgeProps) => {
```

Comment:
```suggestion
const Badge = ({color or type, children}: IBadgeProps) => {
```
Rend le générique
You cannot select a badge, you can choose his color

Reply:
it's was used in a component and when I select the component the color change but fine I can do it like you say the color so the isSelected is used in the parent component color={isSelected ? "":""}